iphone/Maps/Core/Location/MWMLocationManager.mm

```diff
@@ -319,6 +319,7 @@ + (void)setMyPositionMode:(MWMMyPositionMode)mode
     {
     case MWMRouterTypeVehicle: manager.geoMode = GeoMode::VehicleRouting; break;
     case MWMRouterTypePublicTransport:
+    case MWMRouterTypeHelicopter:
```

Comment:
No, it's better to insert it at the end. Please try to understand what the previous code did and decide if it is relevant here.

Reply:
Also, sometimes it is better to leave a warning and come back to fix it later.

Reply:
Is this ok now?